### PR TITLE
Style improvement for empty page

### PR DESCRIPTION
**Pull request by ravikumar1002**
empty page
style improvement
toast
error page

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,9 +1,27 @@
 @import url(./css//main.css);
+/* @import url(react-loader-spinner/dist/loader/css/react-spinner-loader.css); */
 
 .header-wrapper {
   top: 0;
   position: sticky;
   left: 0;
   z-index: 4;
-  background: var(--light-white-bg);
+  background: linear-gradient(63deg, #1c076ff2 2%, #07293ded 100%);
```
Comment (xii-vi) [inline src/App.css:9]: use Variable Name for colors for consistency

File: src/App.js [hunk 2]
```diff
@@ -23,20 +27,24 @@ function App() {
...
           <Header />
         </div>
-        <div style={{ display: "flex" }}>
-          <div>
+        <div className="d-flex">
+          <div className="aside-container">
             <AsideBar />
           </div>
-          <div className="w-100">
+          <div className="w-100" style={{ background: "#fefefe" }}>
```
Comment (xii-vi) [inline src/App.js:38]: avoid inline styling

File: src/components/aside-bar/Aside-bar.jsx [hunk 3]
```diff
@@ -2,9 +2,21 @@ import { Link, NavLink } from "react-router-dom";
 import "./aside-bar.css";
 
 export const AsideBar = () => {
-    const getActiveStyle = ({ isActive }) => ({
-        color: isActive ? "black" : ""
-    });
+    const activeStyle = {
+        color: "white",
+        background: "#616161",
```
Comment (xii-vi) [inline src/components/aside-bar/Aside-bar.jsx:7]: either use hex code or name for color

File: src/components/empty-page/empty.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+.empty-wrapper > p {
+  font-size: 2rem;
+  font-weight: 400;
+  margin: 1rem;
+}
+
+.empty-btn{
+    display: inline-block;
+    padding: 1rem;
+    color: rgba(7, 10, 71, 1);
```
Comment (xii-vi) [inline src/components/empty-page/empty.css:14]: maintain Consistency

**Review by xii-vi — APPROVED** [review]
Left a few sanity comments. 
adding toast is a nice addition
Also, try not to include to much line of code in single PR

**Review by mansipadgelwar — COMMENTED** [review]
Looks Good. You can merge your PR.

**Pull request merged**